ush/rocoto/setup_expt.py

```diff
@@ -30,7 +30,8 @@ def fill_COMROT(host, inputs):
 
     fill_modes = {
         'cycled': fill_COMROT_cycled,
-        'forecast-only': fill_COMROT_forecasts
+        'forecast-only': fill_COMROT_forecasts,
+        'gefs': fill_COMROT_gefs
```

Comment:
Actually, we might want to use -1 as the default (meaning high-res only). For now assume GEFS-resolution only if the number of members >= 0, though we'll probably need a new switch in the future to turn off the high-res instead, since we'll want the option to everything (GFS, GEFS, GDAS, EnKF) together at some point.